lib/secure/simul/adminp.c

```diff
@@ -2,10 +2,8 @@
  * Check whether object is at least an Admin level wizard.
  */
 #include <wizlevels.h>
-
-int wizardp(object ob);
+#include <object_info.h>
 
 int adminp(object ob) {
-    return ((wizardp(ob)) &&
-            (ob->query_wiz_level() >= LEVEL_ADMIN));
-}
\ No newline at end of file
+  return objectp(ob) && efun::object_info(ob, OI_ONCE_INTERACTIVE) && (ob->query_wiz_level() >= LEVEL_ADMIN);
```

Comment:
One liners are nice and all, but I think breaking this return into three lines at the &&'s is better style.

Reply:
Done